start.sh

```diff
@@ -0,0 +1,17 @@
+ #!/bin/sh
+
+ if ! [ -f '/config/config.yaml' ]; then
+    echo "There is no config.yaml! An example is created."
+    cp /application/config.yaml.example /config/config.yaml.example
```

Comment:
```cp /application/config.yaml.example /config/config.yaml```

Reply:
i'm not sure 🤔
if we copy the example direkt to config.yaml, the next time the container is started it tries to run with the example config.
Nobody want this, thats why i copied it to *.example.

special, if you start the container with restart=unless-stopped (which is my default 😁) copying it direct to config.yaml isn't that nice 😉😁

Reply:
Oh I read it wrong, i thought it was supposed to create default config, when it doesn't exist

Reply:
oh, yes and no, it creates a sample config at build time.
i think, depending on the overlay driver the user never see this file 🤔